github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1293,3 +1293,60 @@ periodics:
       - name: pgp-bot-key
         secret:
           secretName: pgp-bot-key
+- name: periodic-kubevirt-e2e-k8s-1.21-sig-performance
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cron: "40 7,15,23 * * *"
+  decorate: true
+  cluster: prow-workloads
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  labels:
+    preset-dind-enabled: "true"
```

Comment:
I think here we would need the label which the service monitor uses to collect the metrics. The labels here get directly propagated to the pods.

Reply:
I will add `group: kubevirtci`

Reply:
According to https://github.com/kubevirt/project-infra/blob/15807c5f709d63fd002cb21e1ed31f0f4ae76452/github/ci/services/prometheus-stack/manifests/stack/production-e2e-workloads/prometheus-federation.yaml#L17 I think you have to add 

```
ci.kubevirt.io/prometheus: ""
```